pkg/cli/cli.go

```diff
@@ -57,5 +65,8 @@ A very simple cli.`,
 		newDeleteCmd(client),
 	)
 
+	rootCmd.PersistentFlags().IntVarP(&log.LogLevel, "verbosity", "v", 0, "level of log verbosity")
+	rootCmd.PersistentFlags().StringVarP(&log.LogFileName, "print", "p", "", "print logs to file")
```

Comment:
Hm, `print` is a bit confusing. I would use another name. What do you think of `logfile` or something like that?

Reply:
Renamed as `--log-file, -l`